mne_bids/tests/test_read.py

```diff
@@ -71,6 +75,10 @@ def test_not_implemented():
 
 
 @requires_nibabel()
+@pytest.mark.filterwarnings(r'ignore:The unit for channel\(s\) STI 001, '
+                            'STI 002, STI 003, STI 004, STI 005, STI 006, '
+                            'STI 014, STI 015, STI 016 has changed from V to '
+                            'NA.:RuntimeWarning:mne')
```

Comment:
this is too long for my taste. I would simply do:

`@pytest.mark.filterwarnings(r'ignore:The unit for channe*.:RuntimeWarning:mne')`

the details are not important here.